.hintrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
README file should professional. Please add the project name, project description, and instructions to set the project up locally. The live demo is not working. You may remove it or add an active live demo link. Please make sure README is clean and professional as it's the first point of view of your project.